src/Microsoft.Health.Dicom.Core.UnitTests/Features/Validation/TagPathValidatorTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Dicom.Core.Exceptions;
+using Microsoft.Health.Dicom.Core.Features.Validation;
+using Xunit;
+
+namespace Microsoft.Health.Dicom.Core.UnitTests.Features.Validation
+{
+    public class TagPathValidatorTests
+    {
+        [InlineData("helloworld")]
+        [InlineData("(0101,2323)")]
+        [InlineData("0101,2323")]
+        [InlineData("(01012323)")]
+        [InlineData("01012323-01012323")]
+        [InlineData("01012323.01012323 45455656")]
+        [InlineData("0101232301012323")]
+        [Theory]
```

Comment:
>[Theory] [](start = 8, length = 8)

nit: I have typically seen the `[Theory]` attribute first, as if to say, "This method is a theory for the following scenarios". Also, some part of me likes to see the smaller line on top of the larger lines lol. But if this is the typical order we use, then nevermind!

Reply:
I do agree that the smaller line below looks strange. However, this is the typical order we use!

---
In reply to: [564825555](https://github.com/microsoft/dicom-server/pull/501#discussion_r564825555) [](ancestors = 564825555)